webapp/src/components/ParcelDetailPage/ParcelDetailPage.js

```diff
@@ -135,9 +136,14 @@ export default class ParcelDetailPage extends React.PureComponent {
               <Grid.Row>
                 <Grid.Column>
                   <h3>{t('asset_detail.bid.title')}</h3>
-                  {bids.map(bid => (
-                    <Bid key={bid.id} bid={bid} isOwner={isOwner} />
-                  ))}
+                  {bids
+                    .filter(
+                      bid =>
+                        (isOwner && isOpen(bid)) || (!isOwner && isActive(bid))
```

Comment:
Question: Is there a name for this?, I mean for the entire condition.

Reply:
I will try